db/seeds.d/75-salt_seeds.rb

```diff
@@ -1,13 +1,2 @@
 # Create feature for Smart Proxy
 Feature.where(:name => 'Salt').first_or_create
```

Comment:
Not for this PR, but there should also be a Foreman plugin DSL to define a feature.